Blasphemous.Framework.Levels/LevelFramework.cs

```diff
@@ -297,14 +298,23 @@ protected override void OnRegisterServices(ModServiceProvider provider)
             new SceneLoader("D17Z01S10_LOGIC", "LOGIC/INTERACTABLES/{0}"),
             new BloodModifier()));
 
-        // Traps
-        provider.RegisterObjectCreator("spikes", new ObjectCreator(
+        // spikes
```

Comment:
Comments should start with an uppercase letter.  This also applies to the comments in the documentation for each modifier, capitalize those as well